sample-declarative-ui/iosApp/iosApp/mokoStringDescExt.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  mokoStringDescExt.swift
+//  iosApp
+//
+//  Created by mdubkov on 02.06.2022.
+//  Copyright © 2022 orgName. All rights reserved.
+//
+
+import Foundation
+import MultiPlatformLibrary
+import mokoMvvmFlowSwiftUI
+
+extension ObservableObject where Self: ViewModel {
+    func stateNullable<T, R>(
+        _ flowKey: KeyPath<Self, CStateFlow<T>>,
+        equals: @escaping (T?, T?) -> Bool,
+        mapper: @escaping (T?) -> R?
+    ) -> R? {
+        let stateFlow: CStateFlow<T> = self[keyPath: flowKey]
+        var lastValue: T? = stateFlow.value
+        
+        var disposable: DisposableHandle? = nil
+        
+        disposable = stateFlow.subscribe(onCollect: { value in
+            if !equals(lastValue, value) {
+                lastValue = value
+                self.objectWillChange.send()
+                disposable?.dispose()
+            }
+        })
+        
+        return mapper(stateFlow.value)
+    }
```

Comment:
why it's required?

Reply:
[mokoMvvmFramework](https://github.com/icerockdev/moko-mvvm/blob/e6023df467cd297ea20f5764b612535929e87a49/mvvm-flow/apple/xcode/mokoMvvmFlowSwiftUI/ViewModelState.swift#L32) does not allow to get a nullable value from CStateFlow, it is necessary to add

Reply:
add issue for this please